src/swap/SwapReviewScreen.tsx

```diff
@@ -119,34 +115,38 @@ export function SwapReviewScreen(props: Props) {
         setShouldFetch(false)
         setFetchError(true)
         dispatch(showError(ErrorMessages.FETCH_SWAP_QUOTE_FAILED))
-        Logger.debug(TAG, 'Error while fetching transactions', error)
+        Logger.debug(TAG, 'Error while fetching swap quote', error)
       },
     }
   )
 
   const submitSwap = () => {
-    // Check for swapInfo prior to submitting swap
-    if (!swapInfo) return
+    // Check for swapResponse prior to submitting swap
+    if (!swapResponse) return
+
     // Analytics for swap submission
     ValoraAnalytics.track(SwapEvents.swap_review_submit, {
       toToken,
       fromToken,
       // TODO: Add fee to total when enabled
-      usdTotal: +divideByWei(swapInfo.unvalidatedSwapTransaction.buyAmount).multipliedBy(
-        swapInfo.unvalidatedSwapTransaction.price
+      usdTotal: +divideByWei(swapResponse.unvalidatedSwapTransaction.buyAmount).multipliedBy(
+        swapResponse.unvalidatedSwapTransaction.price
       ),
-      fee: +divideByWei(swapInfo.unvalidatedSwapTransaction.sellAmount).multipliedBy(
+      fee: +divideByWei(swapResponse.unvalidatedSwapTransaction.sellAmount).multipliedBy(
```

Comment:
In my opinion we should still include the fee's in analytics even if we aren't enabling them, this would give us an idea on what the total fees on all attempted swaps would be.

Reply:
currently this "fee" value is based on some percentage that we have on remote config which may not be what we want to use in the end. isn't it more realistic that if we want to calculate the potential revenue from fees, that we calculate all the swap amounts instead? i'm not sure why we would use this value instead, so i still don't really see the value of keeping this property